lib/galaxy/schema/schema.py

```diff
@@ -3739,10 +3740,17 @@ class PageSummaryBase(Model):
     slug: str = Field(
         ...,  # Required
         title="Identifier",
-        description="The title slug for the page URL, must be unique.",
-        pattern=r"^[^/:?#]+$",
+        description="The identifiying slug for the page URL, must be unique.",
```

Comment:
```suggestion
        description="The identifying slug for the page URL, must be unique.",
```
I think this typo is the cause of the open-api linter failure...